.gitmodules

```diff
@@ -1,9 +1,6 @@
 [submodule "lib/kengdic"]
 	path = lib/kengdic
 	url = git@github.com:scottgigante/kengdic
-[submodule "lib/gTTS-token"]
```

Comment:
Is gTTS-token no longer required?

Reply:
https://github.com/scottgigante/korean-support/commit/11099741bb3b446c225e4019fcd4e6a9cb22f832

Was removed before but was still left in the .gitmodules. Just noticed it's still in one of the readmes, gonna remove it from there too and revise the merge request. Isn't currently being shipped in the download and the [readme](https://github.com/Boudewijn26/gTTS-token/blob/master/README.md) states it's no longer needed.

Reply:
Thanks!